### DISCUSSION TITLE
Powering BlueRetro PS1&PS2 adapter through external charger

### DISCUSSION THREAD

**Discussion by LLLLBX3041** [darthcloud/BlueRetro#1044] [Q&A]
Hi, I noticed a slight improvement in input latency while powering my PS1&PS2 adapter with a phone charger. I'm using a 5V 2.4 A charger. Is it safe to power the adapter with the charger during gameplay, even if my console (PS1) is capable of powering the adapter? Am I using the right charger? How much power does the adapter require to work? Thanks a lot.

**Comment by NosAlu** [comment]
No hay ningun problema 
El ESP32 necesita 5v
Por el amperaje no te preocupes aunque le conectes uno de 4.3A el cargador solo va a liberar el amperaje que necesita el ESP32
Ahora si lo alimentas con mas de 5v si puede dañar

**Reply by LLLLBX3041** [reply]
Thank you! How do you know this? I was wrong about my charger, it is a 5V 2.1 A. Maybe a usb 3.0 charger (5V, 3A) could be better than the one I have (just to be sure the adapter draws all the power it needs to function correctly)? I don't know, maybe amperage matters only in the case of a battery-powered device.

**Reply by NosAlu** [reply]
Mientras el voltaje sea de 5v el amperaje no me afecta
Si el ESP32 no necesita amperaje el cargador no lo libera
Es lo mismo que un cargador de una laptop por ejemplo
Si tenes una notebook que necesita 19v y 2A y tenes un cargador de 19v y 3.41A no hay ningun problema ya que la notebook solo va a consumir lo que necesita...

**Reply by LLLLBX3041** [reply]
Thank you! I feel reassured now!
